src/Nox.Generator/Application/Commands/UpdateCommand.template.cs

```diff
@@ -107,4 +120,25 @@ private async Task UpdateLocalizedEntityAsync({{entity.Name}}Entity entity, {{en
 		_entityLocalizedFactory.UpdateLocalizedEntity(entityLocalized, updateDto);
 	}
 	{{- end }}
-}
\ No newline at end of file
+}
+
+{{- if (entity.OwnedRelationships | array.size) > 0 }}
+
+public class Update{{entity.Name}}Validator : AbstractValidator<Update{{entity.Name}}Command>
+{
+    public Update{{entity.Name}}Validator()
+    {
+		{{- for ownedRelationship in entity.OwnedRelationships }}
+			{{- if ownedRelationship.WithMultiEntity }}
+				{{- relationshipName = GetNavigationPropertyName entity ownedRelationship }}
+				{{- key = ownedRelationship.Related.Entity.Keys | array.first }}
+					{{- if IsNoxTypeCreatable key.Type }}
```

Comment:
Guid is a special case and we should be able to provide items without providing a Guid value as a key, and they should be just created like new entries.
We can have tests for Guid keys for these cases.

Reply:
Have update this and some other validations for Guid